tempo 6 (lar)

marcoscv-work opened the pull request.

jkappler committed:
LPS-66815 Create basic Form Browser Portlet

jkappler committed:
LPS-66815 Create display context and inject it in the request

jkappler committed:
LPS-66815 Create views for form browser

jkappler committed:
LPS-66815 Create browse portlet provider

marcoscv-work committed:
tempo

marcoscv-work committed:
tempo 2

marcoscv-work committed:
goon

jkappler committed:
LPS-66815 Move classes to new internal structure

pavel-savinov committed:
LPS-58672 move missing configuration from portlet properties to config admin

pavel-savinov committed:
LPS-58672 use configuration interface instead of portlet properties

pavel-savinov committed:
LPS-58672 missing language bundle

pavel-savinov committed:
LPS-58672 build lang

marcoscv-work committed:
tempo 3

Commit:
LPS-67383 Create tree view card component

Commit:
LPS-67383 Implement tree view

Commit:
LPS-67383 Add package.json

marcoscv-work committed:
tempo 4 (gulp file)

pavel-savinov committed:
LPS-58672 missing dependencies

pavel-savinov committed:
LPS-58672 update breaking changes

Commit:
LPS-67383 Add icon to card

Commit:
SYNC-1656 Add control panel option to force security mode on mobile devices

Commit:
SYNC-1656 Switch dependency to project

Commit:
SYNC-1656 Baseline

ealonso committed:
LPS-67383 Adds dependency to metal

ealonso committed:
LPS-67383 Split css

ealonso committed:
LPS-67383 Auto SF

marcoscv-work committed:
tempo 5

marcoscv-work committed:
tempo 6 (lar)

marcoscv-work commented:
@antoniopol06 La importación del site template no funciona, por lo que tendrás que aplicar el theme a cualquier site o página.

cc @jbalsas

antoniopol06 commented:
Just started reviewing :) <br><br>:octocat: _Sent from [GH](http://nodegh.io)._

marcoscv-work commented:
@antoniopol06 ! puedes ir a control panel -> site templates -> crear uno nuevo (puedes llamarlo como quieras).
Entras en ese site template y te vas a:

![image](https://cloud.githubusercontent.com/assets/7963804/17357980/add36262-595f-11e6-8beb-c182954ae8f0.png)

Ahí haces una importación del lar, y de esa forma si te importa los contenidos!!!

juliocamarero committed:
LPS-58672 SF

mdelapenya committed:
LPS-65384 Add a test verifying the bug

ealonso committed:
LPS-67383 Uses item selector to open the dialog

ealonso committed:
LPS-67383 Creates new method to return the journal folder tree as a JSONObject

Commit:
LPS-67383 Add selected and disabled nodes

Commit:
LPS-67383 Change folderId to id

mdelapenya committed:
LPS-65384 Use proper tag value

mdelapenya committed:
LPS-65384 Add tests covering the changes

mdelapenya committed:
LPS-65384 Remove spaces from strip

Commit:
LPS-67448 Web Form: Syntax Error when using simple quotes in Validation Error Message field

Commit:
LPS-67448 Using HtmlUtil.escapeJS() in javascript codepart

Commit:
Main.js using metal-dom

Commit:
Add focus class to search hover

Commit:
SF

juliocamarero committed:
LPS-67120 Deprecate old configuration classes

juliocamarero committed:
LPS-58672 simplify listeners logic

adolfopa committed:
LPS-67508 Always return the last approved version

Commit:
Click close button when input is empty works

marcoscv-work committed:
show-site-name-default to false

gregory-bretall committed:
LPS-67393 Strip the HTML out from a comment's body when making it a subject when checking if the comment is too many characters or if it needs to be abbreviated

Commit:
Blur works

juliocamarero committed:
LPS-58672 move class to another module so that it is not installed or bundled by default but people can still use... Do we really want to keep this?

gregory-bretall committed:
LPS-67393 Strips HTML tags from a comment's subject and adds an ellipsis if the comment's text is over 50 characters

brianchandotcom committed:
LPS-58672 auto SF

brianchandotcom committed:
LPS-58672 Hugo SF

brianchandotcom committed:
LPS-58672 Hugo SF

brianchandotcom committed:
LPS-58672 Hugo SF

juliocamarero committed:
LPS-58672 semantic versioning

brianchandotcom committed:
LPS-58672 auto SF

brianchandotcom committed:
LPS-58672 Clarity

brianchandotcom committed:
LPS-58672 SF

brianchandotcom committed:
LPS-58672 SF

brianchandotcom committed:
LPS-58672 Hugo SF

BryanEngler committed:
LPS-67393 SF

jonathanmccann committed:
LPS-67393 Apply logic to 'updateDiscussionMessage' as well and extract it out to reduce duplicated code

Commit:
LPS-67304 - Update DDM date field nullable check to always be nullabe for DDM fields

anthony-chu committed:
LRQA-27163 Add method to return a percentage of a given value

liuxingye committed:
LRQA-27146 Use Blogs macro that incorporates auto-save

jbalsas committed:
LPS-67383 Do not render unexpanded subtrees

jbalsas committed:
LPS-67383 Rename to CardsTreeView

jbalsas committed:
LPS-67383 No need for trailing comma in single selection mode

jbalsas committed:
LPS-67383 Use default state events instead of custom callbacks

jbalsas committed:
LPS-67383 SF

jbalsas committed:
LPS-67383 Params defined in the .render template automatically become state

jbalsas committed:
LPS-67383 Improve jsdoc

marcoscv-work committed:
SF

anthony-chu committed:
LRQA-27224 Remove empty string type before focusing on input field

marcoscv-work committed:
JS SF

GinsonRen committed:
LRQA-27014 Add new path for entry title under list style

GinsonRen committed:
LRQA-27014 Add new macro to view web content display style

GinsonRen committed:
LRQA-27014 Add new case to view web content display style

marcoscv-work committed:
Adding sing-in theme style

marcoscv-work committed:
Icon removed

marcoscv-work committed:
link state styles

Commit:
LPS-67383 Correct selectedNode checking

Commit:
LPS-67383 Make tabable only the first node

Ithildir committed:
LPS-67658 Add tasks in the root project to extract Gradle API jars

Ithildir committed:
LPS-67658 Ignore Gradle root project build directory

Ithildir committed:
LPS-67658 These plugins must work with Gradle 2.5+

Ithildir committed:
LPS-67658 These plugins must work with Gradle 2.14+

koor4356 committed:
LRQA-26979 Move tests from CPMydashboard to MySites testcase

marcoscv-work committed:
hader fix

marcoscv-work committed:
new simpler lar

pavel-savinov committed:
LPS-62977 check for hide default success message before add it to avoid duplication in ajax request

Commit:
LPS-67650 Add new attribute and setter/getter pair to PDC

Commit:
LPS-67650 Add new graph operations to the StagedModelRepository

pavel-savinov committed:
LPS-58672 fix integration tests

pavel-savinov committed:
LPS-58672 missing dependency

pavel-savinov committed:
LPS-58672 SF

Commit:
LPS-67650 Use layout id from PDC and populate

Commit:
LPS-67650 Fix cloning of StagedLayoutSetImpl

Commit:
LPS-67650 Missing overrides from StagedLayoutSetImpl

Commit:
LPS-67650 Add new StagedLayoutSetStagedModelRepository class

Commit:
LPS-67650 Semver, project

Commit:
LPS-67650 Better to return privateLayout as uuid

Commit:
LPS-67650 Apply to Bookmarks

Commit:
LPS-67650 Apply to DDL

pat270 committed:
LPS-67383 Source Format

Ithildir committed:
LPS-67658 Need "compileOnly" to keep dependencies out of "compile"

koor4356 committed:
LRQA-27134 Update, remove unnecessary, and add new macro

koor4356 committed:
LRQA-27134 Update and add a new test

anthony-chu committed:
LRQA-27134 Allow multiple instances of locator

anthony-chu committed:
LRQA-27134 Iterate with list for reusability

koor4356 committed:
LPS-67568 Enable RSS and RSS ADT acceptance tests

ctampoya committed:
LRQA-26387 Edit use case ticket to correct LPS

ctampoya committed:
LRQA-26387 Declare workflow task variable

ctampoya committed:
LRQA-26387 Do not need to navigate back to Kaleo Forms portlet

Commit:
LPS-67383 Update package.json  version

Commit:
LPS-67383 Move click handler to card

jkappler committed:
LPS-51377 Check for image size when uploading file

jorgediaz-lr committed:
LPS-67702 Fix invalid friendly urls

CAustin582 committed:
LPS-67620 Avoid defining directories with ${basedir} so that the script will work when invoked from other locations

Commit:
LPS-67695 Add unknown columns to Marketplace_App table

ctampoya committed:
LRQA-26387 Use workflow macro to view task details

ctampoya committed:
LRQA-26387 Go to workflow edit screen

ctampoya committed:
LRQA-26387 View second workflow submission from same asset

ctampoya committed:
LRQA-26387 Add second process record after assigning the first record

ctampoya committed:
LRQA-26218 Navigate to My Workflow Tasks

ctampoya committed:
LRQA-26387 Remove Kaleo form portlet macros

ctampoya committed:
LRQA-26387 Consistently use all upper case letters

lesliewong92 committed:
LRQA-26692 Rename file before failing build

ctampoya committed:
LRQA-26387 Use correct variable

anthony-chu committed:
LRQA-27224 Use type to avoid appending the new string onto the old in a simple text field

ZoltanTakacs committed:
LRDOCS-2194 Use Base64 encoded image instead of an external src

ZoltanTakacs committed:
LRDOCS-2194 Regen

ctampoya committed:
LRQA-26387 SF

Ithildir committed:
LPS-67658 Use latest gradle-bundle-plugin

Ithildir committed:
LPS-67658 cache

lesliewong92 committed:
LRQA-27163 Simplify

Commit:
LRDOCS-2881 Edit Javadoc

CAustin582 committed:
LPS-67620 ${portal.dir} might be undefined at this point

Ithildir committed:
LPS-67658 Start adding change logs

brianchandotcom committed:
LPS-58672 Typo

brianchandotcom committed:
LPS-58672 Regen lang

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/journal/.gitrepo'.

pei-jung committed:
LPS-67712 Back button to navigate back to component permissions page is not responsive

brianwulbern committed:
LRQA-27261 Grab siteName based on test branch

brianwulbern committed:
LRQA-27146 Remove unused Tag macro code blocks

brianwulbern committed:
LRQA-27146 Remove conditional entrySubtitle not in original test

lesliewong92 committed:
LRQA-27108 Run subrepository targets given a subrepository name

michaelhashimoto committed:
LRQA-27281 Print a jstack if before liferay starts it times out

brianwulbern committed:
LRQA-27146 Line-spacing SF

brianchandotcom committed:
Sync

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/plugins-admin/.gitrepo'.

Commit:
LPS-67722 Download an attachment should be a get method.

brianchandotcom committed:
LPS-66815 SF

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/forms-and-workflow/dynamic-data-lists/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/marketplace/.gitrepo'.

brianchandotcom committed:
LPS-67658 prep next, use latest

brianchandotcom committed:
LPS-67658 cache

brianwulbern committed:
LRQA-27257 Move and rename localization locator

brianwulbern committed:
LRQA-27255 Update variable name

brianwulbern committed:
LRQA-27269 Remove backslashes from search string

Commit:
LPS-67650 Apply to AssetLink

Commit:
LPS-67650 Apply to DDMDataProviderInstanceStagedModelRepository

robertoDiaz committed:
LPS-67698 Add missing fields

robertoDiaz committed:
LPS-67698 Do not use the ModelHintsUtil since it could return a wrong value for this upgrade

Commit:
LPS-67383 Align cards correctly

brianwulbern committed:
LRQA-27014 Lower the test priority

hhuijser committed:
LPS-67352 Add custom check 'MultipleJavadocCheck'

hhuijser committed:
LPS-67352 SF

hhuijser committed:
LPS-67352 Add custom check 'MissingAuthorCheck'

hhuijser committed:
LPS-67352 Remove existing logic

hhuijser committed:
LPS-67352 Fix tests

hhuijser committed:
LPS-67352 Use a set so we don't add the same file multiple times and therefore get the same checkstyle errormessage multiple times

Commit:
LPS-67383 baseline

Commit:
LRQA-27282 Add pause into test before navigating to publishing

inacionery committed:
LPS-67729 Fix package and imports

inacionery committed:
LPS-67729 Update Gradle to reference projects and add missing references

codyhoag committed:
LPS-67658 Edit Change Logs

codyhoag committed:
LPS-67658 Use "the" to emphasize the subject, for consistency

Commit:
LPS-67650 Unused variable

adolfopa committed:
LPS-67591 Avoid NPE for files in the root dir

When the ZIP contains relative file names, entries at the root will
return `null` as their parent path.  Treat the `null` path as equivalent
to `/` to avoid a NPE exn.

Commit:
LPS-67698 SF

anthony-chu committed:
LRQA-27014 Rename

pyoo47 committed:
LRQA-27283 SEMVER

Ithildir committed:
LPS-58672 Add missing build.gradle

adolfopa committed:
LPS-67393 Reduce indentation and nesting

brianchandotcom committed:
artifact:ignore marketplace-service 2.1.11 prep next

brianchandotcom committed:
artifact:ignore web-form-web 1.0.2 prep next

brianchandotcom committed:
artifact:ignore plugins-admin-web 1.0.5 prep next

Ithildir committed:
LPS-67694 Check the snapshot staleness during configuration instead

This way we can completely disable the install/uploadArchives task and its dependencies. It will be faster to call "gradlew uploadArchives -PsnapshotIfStale" if the snapshot is not stale and doesn't need to be published.

brianchandotcom committed:
LPS-67352 prep next, use latest

brianchandotcom committed:
LPS-67352 cache

brianchandotcom committed:
LPS-67695 SF

brianchandotcom committed:
artifact:ignore marketplace-service 2.1.12 prep next

brianchandotcom committed:
artifact:ignore message-boards-web 1.1.1 prep next

brianchandotcom committed:
Sync

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/collaboration/document-library/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/collaboration/item-selector/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/collaboration/message-boards/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/plugins-admin/.gitrepo'.

brianchandotcom committed:
LPS-67650 Rename

Ithildir committed:
LPS-67352 Update change logs

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/marketplace/.gitrepo'.

brianchandotcom committed:
LPS-67650 as used

brianchandotcom committed:
LPS-67650 simplify

brianchandotcom committed:
LPS-67650 SF

Ithildir committed:
LPS-67694 Update change log

gamerson committed:
LPS-65477 add portal kernel since compileOnly dependencies are not transitive

gamerson committed:
LPS-65477 switch to gradle-plugins 2.0.10 since it is required to support compileOnly properly

Ithildir committed:
LPS-65477 Add change log

Ithildir committed:
LPS-67694 Better change log titles

brianchandotcom committed:
LPS-67650 Hugo SF, avoid ".stream()."

codyhoag committed:
LPS-58672 Edit breaking change

brianchandotcom committed:
LPS-67650 Hugo SF, avoid chaining inside a stream

Ithildir committed:
LPS-67694 Use Gradle to download the latest artifact, not Nexus

See http://stackoverflow.com/a/25951599/799771

kenjiheigel committed:
LRQA-66736 Ignore this error, as it only occurs sporadically on JBoss/Wildfly

brianchandotcom committed:
LPS-67650 Hugo SF - avoid chaining in streams

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/collaboration/bookmarks/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/forms-and-workflow/dynamic-data-lists/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/forms-and-workflow/dynamic-data-mapping/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/asset/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/export-import/.gitrepo'.

brianchandotcom committed:
SYNC-1656 lang keys

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/layout/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/sync/.gitrepo'.

brianchandotcom committed:
SYNC-1656 thai

brianchandotcom committed:
LPS-65477 prep next, use latest

brianchandotcom committed:
LRQA-27108 SF

brianchandotcom committed:
LPS-67393 SF

brianchandotcom committed:
LPS-67694 prep next, use latest

brianchandotcom committed:
LPS-67694 cache

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/forms-and-workflow/dynamic-data-mapping/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/sync/.gitrepo'.

kenjiheigel committed:
Revert "LPS-67650 Hugo SF - avoid chaining in streams"

This reverts commit 0c9303c6425faeaecc10a38fb7289a2b3b18aa97.

brianchiu committed:
Revert "LPS-67469 Move Private Messaging outside of Collaboration because it should be an independent marketplace lab app"

This reverts commit 992078c51c8c7fdb5234dc17e8ce6af492628b49.

codyhoag committed:
LPS-65477 Edit Change Logs

brianchandotcom committed:
LPS-67650 temp for now

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/export-import/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/collaboration/social/.gitrepo'.

brianchandotcom committed:
LPS-67383 SF

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/journal/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/journal/.gitrepo'.

brianchandotcom committed:
LPS-62977 make it protected so others can override it

brianchandotcom committed:
LPS-62977 auto SF

brianchandotcom committed:
LPS-62977 baseline

brianchandotcom committed:
LPS-62977 Regen

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/asset/.gitrepo'.

marcoscv-work committed:
new lar

marcoscv-work committed:
Layout renamed

marcoscv-work committed:
new 50/50 limited layout
